docs/osm2pgsql-replication.1

```diff
@@ -174,10 +202,13 @@ Database server host name or socket location
 Database server port
 
 .TP
-\fB\-\-prefix\fR PREFIX
+\fB\-p\fR PREFIX, \fB\-\-prefix\fR PREFIX
 Prefix for table names (default 'planet_osm')
 
+.SH AUTHORS
+.B <<UNSET \-\-project\-name OPTION>>
+was written by <<UNSET \-\-author OPTION>> <<<UNSET \-\-author_email OPTION>>>.
 .SH DISTRIBUTION
-The latest version of osm2pgsql\-replication may be downloaded from
+The latest version of <<UNSET \-\-project\-name OPTION>> may be downloaded from
```

Comment:
Please make sure these four lines remain unchanged: use the project-name option and remove the authors section manually